android/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         google()
         jcenter()
         mavenCentral()
-        maven { url "https://dl.bintray.com/terl/lazysodium-maven" }
+        maven { url "http://github.com/terl/lazysodium-android" }
```

Comment:
I wonder if theres a way to not import a github url! I left our conversation thinking about how it may be slow

Reply:
Would this be better? `https://mvnrepository.com/artifact/com.goterl.lazycode/lazysodium-java`

Reply:
we need lazysodium-android

Reply:
Oops ya, I just grabbed that from maven central as an example but probably would have used it. Thanks for catching that

Reply:
check if it works, not sure if it will i get a 404

Reply:
Lazy sodium 5.x should be pulled from maven central without a custom import. But if it isn't working I will dig in at a later date because I am not supposed to be commenting today